### DISCUSSION TITLE
タグランキングについて

### DISCUSSION THREAD

**Discussion by eduidl** [increments/qiita-discussions#260] [要望・アイデア・不具合]
現在のタグランキングが以下の通りですが、3~6位のようにごく少数のユーザが大量投稿しているだけのタグだったり、「初心者」や「coding」といういまひとつランキングにあっても困るもの（codingってQiitaの記事大概そうでしょ…）だったりが半数を締めていてあまり情報量がないなと感じています。
そもそもタグランキングの目的を自分が勘違いしているだけかもしれませんが、自分は最近の技術トレンドを知るためのものだと思っていたので、この現状はタグランキングとしての役割を果たせているのかな？と疑問が付きます。
速報性や正確性は落ちるかもしれませんが、ホワイトリストを用意するなどして、もう少しそれっぽいランキングが見たいなと思います。

![image](https://user-images.githubusercontent.com/25898373/180655558-fb88eb49-3f0e-4922-b1f8-c8f0f7846b51.png)

**Comment by tanaka-asami415** [comment]
@eduidl 
ご意見ありがとうございます🙏
[トップページの情報整理 #172](https://github.com/increments/qiita-discussions/issues/172) でタグランキングについても見せ方を検討中で、ベータ版として公開予定です。
その際は是非フィードバック等でご意見をお聞かせいただけるとありがたいです🙏✨

**Comment by mickie895** [comment]
単純にそのタグを「投稿した人の数」でランキングにしても良いと思います。
技術トレンドは一人では作れないでしょうし。

**Reply by tanaka-asami415** [reply]
@mickie895 
ご意見ありがとうございます🙏
現在のランキングの状況は運営も問題に感じておりますので
今回頂いたご意見や、その他の方法も含めて検討させていただきます！💪

**Comment by nkay0** [comment]
別にスレッドを立てたほうがいいかもしれませんが、この3~6位のタグを投稿している少数というか1人のユーザーの行動が、Qiitaの利用規約や平均的なユーザーのコモンセンスに照らして問題がないとは思えないのですが、Qiita運営としてはどうなんでしょうか。

**Reply by eduidl** [reply]
別にスレッド立ててください

**Reply by tanaka-asami415** [reply]
@nkay0 
恐れ入りますが、コミュニティガイドラインに沿って対応をしておりますので、個別に回答はいたしかねます。
利用規約やコミュニティガイドライン違反の報告につきましては、通報機能をご利用いただけますと幸いです🙇

**Reply by nkay0** [reply]
はい

**Comment by kyntk** [comment]
@eduidl @mickie895 
フィードのサイドバーに表示されているタグランキングの集計方法を変更しました！
ご意見としていただいていた、投稿ユーザー数を集計するように変更しました。
https://qiita.com

**Reply by eduidl** [reply]
良くなったと思いました
